lib/ScanTools.js

```
@@ -15,6 +15,32 @@ function getNext(links) {
     return links.find(link => link.rel === "next"); // returns undefined when not found
 }
 
+function setUiUrl(results, scanId, scanConfigId) {
+
+    let orgToken = "";
+    let appId = "";
+
+    for(let i=0; i<Object.keys(results).length; i++){
+
+        try{
+            orgToken = results.data.data[i].insight_ui_url;
+
+            if(orgToken) {
+                orgToken = orgToken.split("#");
+                orgToken = orgToken[0];
+    
+                appId = results.data.data[0].app.id;
+                return `${orgToken}#/apps/${appId}/configuration/${scanConfigId}/scan/${scanId}`;
+            }
+
+        } catch(err){
```

Generally we probably shouldn't swallow errors as it's not helpful to anyone trying to use the logs to figure out issues. In the type of case where we don't really care if an error occurs we could log debug. For this specific case id go with defensive coding by checking if elements we expect to exist actually do exist before using them, rather than surrounding everything in try/catch

e.g if results.data && results.data.data, this can be a bit tedious, in later versions of node there's a feature called "optional chaining" that makes this much easier, but pretty sure it's not available in the version we're using.